### PR TITLE
Added note that Opus support is High Sierra-only

### PR DESCRIPTION
**Pull request by daytonlowell**

### CODE REVIEW

**Comment by Schweinepriester** [comment]
Would be nice if the note was more future-proof, for example „Supported on macOS 10.13 High Sierra and later“ 0:-)

File: features-json/opus.json [hunk 1]
```diff
@@ -184,8 +184,8 @@
       "9.1":"n",
       "10":"n",
       "10.1":"n",
-      "11":"y",
-      "TP":"y"
+      "11":"y #1",
+      "TP":"y #1"
```
Comment (dstorey) [inline features-json/opus.json]: This should probably be listed as partial. IE got downgraded due to not being included in all OS versions, e.g. https://caniuse.com/#feat=http2 and https://caniuse.com/#feat=mediasource

Comment (daytonlowell) [reply features-json/opus.json]: Fair. If we do that, HEVC and FLAC will need the same treatment.

Comment (Schweinepriester) [reply features-json/opus.json]: Yeah, how to handle OS-depended support is still inconsistent. I asked in #3572, but maybe this should be discussed on its own to find a solution for all occurrences?

Comment (Fyrd) [reply features-json/opus.json]: I'd say as long as the browser is used on an OS version used by a significant amount of users it should be partial support. (as to what's a "significant" amount, ah...that gets tricky)

Comment (Fyrd) [reply features-json/opus.json]: In this case MacOS Sierra still has lots of usage (even higher than High Sierra last month) I agree all these should be partial support, @daytonlowell.

**Comment by Fyrd** [comment]
@daytonlowell Thanks!

**Comment by slokhorst** [comment]
This is not true.. e.g. it doesn't work here: https://hpr.dogphilosophy.net/test/index.php

revert: #4146

**Pull request merged**